app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -127,5 +127,15 @@ def sync_root_tenant
     def create_cloud_tenant(options)
       CloudTenant.create_cloud_tenant(self, options)
     end
+
+    def self.connected_provider_types
+      types = ManageIQ::Providers::CloudManager.pluck(:type).uniq
+      vcloud_included = types.delete('ManageIQ::Providers::Vmware::CloudManager').present?
```

Comment:
I've used the same approach as implemented in EmsCluster: https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_cluster.rb#L338
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper.rb#L1552
Related PR: https://github.com/ManageIQ/manageiq/pull/2708
I agree we need to find more proper way. It should cover cluster naming also.